src/App.tsx

```diff
@@ -81,48 +82,46 @@ function useInterval(callback: () => void, delay: number) {
 }
 
 const App: React.FC = () => {
-  const [timeEstimates, setTimeEstimates] = useState<Array<number>>([]);
+  const [etds, setEtds] = useState<Array<ETD>>([]);
 
   const updateTrains = () => {
-    allEstimatesForEveryTrain("woak", "s").then(
-      (allEstimates: Array<number>) => {
-        setTimeEstimates(allEstimates);
-      }
-    );
+    allEtdsForEveryTrain("woak", "s").then((allEstimates: Array<ETD>) => {
+      setEtds(allEstimates);
+    });
   };
 
   // initial fetch without delay!
   useEffect(updateTrains, []);
 
   useInterval(updateTrains, 40 * 1000);
 
-  console.log("Time estimates are", timeEstimates);
-
-  const formattedTimeEstimates = timeEstimates.map(
-    (time: number, i: number) => {
-      const formattedString = `${time} minutes`;
-      const bartBarWidth = `calc(100% - ${5 * time}px)`;
-      return (
-        <div className="bart-row" key={i}>
-          <span className="time">{formattedString}</span>
-          <div className="bart-bar-background">
-            <div
-              className="bart-bar"
-              style={{
-                width: bartBarWidth
-              }}
-            >
-              <img
-                className="bart-icon"
-                src={happybart}
-                alt="bart, beep beep"
-              />
-            </div>
+  console.log("ETDs are", etds);
+
+  const getMinutesFromETD = (etd: ETD) => {
```

Comment:
This helper can be defined outside this scope. 

1. It may be useful in other places, so no reason to scope it only within this function call.
2. It'll get created everytime react renders this component (not a big deal perf wise, but something to keep in mind)